server/src/domain/job/job.service.ts

```diff
@@ -145,6 +145,7 @@ export class JobService {
     await this.jobRepository.queue({ name: JobName.QUEUE_GENERATE_THUMBNAILS, data: { force: false } });
     await this.jobRepository.queue({ name: JobName.CLEAN_OLD_AUDIT_LOGS });
     await this.jobRepository.queue({ name: JobName.LIBRARY_QUEUE_SCAN_ALL, data: { force: false } });
+    await this.jobRepository.queue({ name: JobName.MOVE_HISTORY_CLEANUP });
```

Comment:
Seeing as a bad move may make the original file unavailable for downloads and other jobs, should we run this cleanup task more regularly?

Reply:
This applies to the task that repairs bad moves - this MOVE_HISTORY_CLEANUP may not be that?

Reply:
This Job is just to delete successful moves from the DB, so that it doesn't become too large. Technically, this isn't even necessary.

Reply:
The actual fixing of bad moves and re-linking of lost files still needs to be done for the most part